composer.json

```diff
@@ -19,13 +19,18 @@
     "chunks"
   ],
   "require": {
-      "php": ">=5.3"
+      "php": ">=5.4"
   },
   "require-dev": {
     "mikey179/vfsstream": "v1.2.0",
     "league/phpunit-coverage-listener": "~1.1",
     "fabpot/php-cs-fixer": "~2.2",
-    "phpunit/phpunit": "4.*"
+    "phpunit/phpunit": "4.*",
+    "mongodb/mongodb": "^1.4.0",
+    "ext-mongodb": "*"
```

Comment:
maybe add these to suggestions instead? They are optional

Reply:
I added them to "require-dev" as they would be needed to run tests in this repo and consumers do not install these dependencies (only "require"). Furthermore I additionally added "mongodb/mongodb" under "suggest" as - like you noted - this is optional from a consumer perspective. 

Does this make sense?